modules/dxp/apps/portal-search-tuning/portal-search-tuning-synonyms-web/src/main/java/com/liferay/portal/search/tuning/synonyms/web/internal/index/SynonymSetIndexReaderImpl.java

```diff
@@ -78,13 +78,6 @@ public List<SynonymSet> search(SynonymSetIndexName synonymSetIndexName) {
 			searchSearchResponse.getSearchHits());
 	}
 
-	@Reference(unbind = "-")
-	protected void setSearchEngineAdapter(
```

Comment:
@W2jx This change should be put in a separate commit "remove duplicate reference", or I would prefer the word "redundant" over "duplicate" because the field is already referenced.